app/src/test/java/org/mozilla/fenix/library/historymetadata/controller/HistoryMetadataGroupControllerTest.kt

```diff
@@ -188,6 +188,7 @@ class HistoryMetadataGroupControllerTest {
     }
 
     @Test
+    @Ignore("Intermittent test: https://github.com/mozilla-mobile/fenix/issues/25167")
```

Comment:
Since there's some requested changes in this PR, can you land the commit for these intermittent test in a separate PR. Assuming these tests are running intermittently regardless of the work here, it can very well affect other CI runs. So, let's just get this in ASAP and not have it be dependent on this PR.

Reply:
#25270

I'll add a comment to the issue tracking these as well